sonatype/src/main/scala/org/typelevel/sbt/TypelevelSonatypePlugin.scala

```diff
@@ -35,17 +35,19 @@ object TypelevelSonatypePlugin extends AutoPlugin {
   import autoImport._
 
   override def buildSettings =
-    Seq(tlSonatypeUseLegacyHost := true) ++
-      addCommandAlias(
-        "tlRelease",
-        mkCommand(
-          List(
-            "reload",
-            "project /",
-            "+mimaReportBinaryIssues",
-            "+publish",
-            "tlSonatypeBundleReleaseIfRelevant"))
-      )
+    Seq(
+      tlSonatypeUseLegacyHost := true,
+      autoAPIMappings := true
```

Comment:
There's no reason this is controversial, right?

Reply:
"This requires that dependencies have this information in its metadata" -- I don't know the consequences if they don't.  It just doesn't link, right?

Reply:
Yep, I assume so.